CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Screen API Coordinator/GiniScreenAPICoordinator+Camera.swift

```diff
@@ -183,13 +183,12 @@ extension GiniScreenAPICoordinator: CameraViewControllerDelegate {
 
     func showNextScreenAfterPicking(pages: [GiniCapturePage]) {
         let visionDocuments = pages.map { $0.document }
-        if let documentsType = visionDocuments.type {
-            switch documentsType {
-            case .image:
-                showReview()
-            case .qrcode, .pdf:
-                showAnalysisScreen()
```

Comment:
[Change request] After your removal of the case .pdf:
- ` Import of the PDF`  from the Camera screen got broken 🙈